resources/function_help/json/is_selected

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "is_selected",
+  "type": "function",
+  "description": "Returns if a feature is selected. If called with no parameters checks the current feature.",
+  "arguments": [
+        {"arg":"feature","description":"The feature which should be checked for selection"},
+        {"arg":"layer","description":"The layer (or its id or name) on which the selection will be checked"}
```

Comment:
Check out the newly introduced array _to_string help - there's a proper way to mark optional arguments in the help so that there are styled correctly